bugbot/utils.py

```diff
@@ -765,3 +765,28 @@ def is_keywords_removed_by_bugbot(bug: dict, keywords: Iterable) -> bool:
         if change["field_name"] == "keywords"
         for keyword in keywords
     )
+
+
+def get_bug_bugdash_url(component, tab_name: str) -> str:
+    """
+    Generate bugdash URL from a component.
+
+    Args:
+        component: The component as a ComponentName object.
+        tab_name: Specifies the tab of the component.
+
+    Returns:
+        Bugdash URL for the component.
+    """
+    # Bugdash uses a single colon instead of a double colon to prefix the product name.
+    encoded_component = quote_plus(f"{component.product}:{component.name}")
+
+    valid_tabs = ["triage", "important", "stalled", "reo", "tracked"]
+    tail = ""
+
+    if tab_name and tab_name in valid_tabs:
+        tail = "#tab." + tab_name
+
+    url = "https://bugdash.moz.tools/?component=" + encoded_component + tail
+
+    return url
```

Comment:
I suggest using f-string to structure the compleat URL at once:
```suggestion
    return f"https://bugdash.moz.tools/?component={encoded_component}#tab.{tab_name}"
```